examples/hello_world_plugin/README.md

```diff
@@ -36,8 +36,9 @@ Now you can run `gz sim` with the name of the resultant library file (without th
 or the file extension, i.e., libHelloWorldPlugin.so -> HelloWorldPlugin):
 
 ~~~
-gz sim --render-engine HelloWorldPlugin
+gz sim -v 4 --render-engine HelloWorldPlugin empty.sdf
```

Comment:
Is verbose needed here for the hello world example?

Reply:
not really, I removed it in [4145113](https://github.com/gazebosim/gz-rendering/pull/892/commits/41451131cfe91f920af537bd280bc5fb5558f4ac). Originally added so that user can check console output to make sure the plugin is loaded but just realized it should still print out an obvious red error msg if plugin is not found without the -v 4 arg.